js/client-teambuilder.js

```diff
@@ -1062,7 +1062,14 @@
 				buf = '<div class="pad"><button name="back"><i class="fa fa-chevron-left"></i> List</button> <input class="textbox teamnameedit" type="text" class="teamnameedit" size="30" value="' + BattleLog.escapeHTML(this.curTeam.name) + '" /> <button name="saveImport"><i class="fa fa-upload"></i> Import/Export</button> <button name="saveImport" class="savebutton"><i class="fa fa-floppy-o"></i> Save</button></div>';
 				buf += '<div class="teamedit"><textarea class="textbox" rows="17">' + BattleLog.escapeHTML(Storage.exportTeam(this.curSetList)) + '</textarea></div>';
 			} else {
-				buf = '<div class="pad"><button name="back"><i class="fa fa-chevron-left"></i> List</button> <input class="textbox teamnameedit" type="text" class="teamnameedit" size="30" value="' + BattleLog.escapeHTML(this.curTeam.name) + '" /> <button name="import"><i class="fa fa-upload"></i> Import/Export</button></div>';
+				buf = '<div class="pad"><button name="back"><i class="fa fa-chevron-left"></i> List</button> ';
+				buf += '<input class="textbox teamnameedit" type="text" class="teamnameedit" size="30" value="' + BattleLog.escapeHTML(this.curTeam.name) + '" /> ';
+				buf += '<button name="import"><i class="fa fa-upload"></i> Import/Export</button> ';
+				buf += '<form style="display:inline" method="post" action="https://pokepast.es/create" target="_blank">';
+				buf += '<input type="hidden" name="title" value="' + BattleLog.escapeHTML(this.curTeam.name) + '">';
+				buf += '<input type="hidden" name="author" value="' + BattleLog.escapeHTML(app.user.get('name')) + '">';
+				buf += '<button type="submit"><input type="hidden" name="paste" value="' + BattleLog.escapeHTML(Storage.exportTeam(this.curSetList)) + '">';
+				buf += '<i class="fa fa-upload"></i> Export to PokePaste</button></form></div>';
```

Comment:
```suggestion
				buf += '<input type="hidden" name="paste" value="' + BattleLog.escapeHTML(Storage.exportTeam(this.curSetList)) + '">';
				buf += '<button type="submit"><i class="fa fa-upload"></i> Export to PokePaste</button>';
				buf += '</form></div>';
```

You still have an input element inside the button.

I looked it up and apparently it's allowed, but I think it makes the button less readable not to all be on one line.

Reply:
I'm also pretty concerned by the `paste` property of the form. You're storing it here, which means it will only be updated by `updateTeamView()` – not all team edits call `updateTeamView()`, so this form could easily send an outdated team.

The easiest way to fix this is to leave these three inputs blank in the HTML, and only fill them out when the button is clicked and an `onSubmit` JavaScript handler is run.